assignment_04/public/js/major.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(function() {
+    $.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        }
+    });
+    // Handle form submission to create a new major
+    $('#createMajorForm').submit(function(e) {
+        e.preventDefault();
+        $.ajax({
+            type: 'POST',
+            url: '/major/store',
+            data: {
+                name: $('#majorName').val(),
+            },
+            success: function(response) {
+                if (response.success) {
+                    $('#majorsList').append(`
+                                            <tr>
+                                                <td>${response.major.id}</td>
+                                                <td>${response.major.name}</td>
+                                                <td>
+                                                    <button class="deleteMajor btn btn-danger btn-sm" data-id="${response.major.id}">Delete</button>
+                                                </td>
+                                            </tr>
+                                            `);
+                    $('#createMajorForm')[0].reset();
+                }
+            },
```

Comment:
How about error condition? 
Please make validation and print error message when validation fail.